app/components/stt-config.tsx

```
@@ -0,0 +1,75 @@
+                </option>
+              ))}
+        </Select>
+      </ListItem>
+      {props.sttConfig.engine === "Web Speech API" && !isFirefox() && (
+        <ListItem title="语言选择">
+          <Select
+            value={props.sttConfig.language}
+            onChange={(e) => {
+              props.updateConfig(
+                (config) => (config.language = e.currentTarget.value),
+              );
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)